src/compile/mark/text.ts

```diff
@@ -81,7 +81,8 @@ export namespace text {
     if (model.has(TEXT)) {
       if (contains([QUANTITATIVE, TEMPORAL], model.fieldDef(TEXT).type)) {
         const format = model.config().mark.format;
-        extend(p, formatMixins(model, TEXT, format));
+        const def = formatMixins(model, fieldDef, format, model.config().mark.shortTimeLabels);
```

Comment:
Why don't put this line in `formatMIxinsText`:

```
const def = formatMixins(model, fieldDef, format, model.config().mark.shortTimeLabels);
```

We don't really need `def` anywhere else here.